src/rpc/RPCHelpers.cpp

```diff
@@ -21,10 +21,12 @@
 
 #include "data/BackendInterface.hpp"
 #include "data/Types.hpp"
+#include "rpc/CredentialHelpers.hpp"
```

Comment:
RPCHelpers.hpp and RPCHelpers.cpp only have headers change. Do we still need to update these two files in this PR?